kong/tracing/propagation.lua

```diff
@@ -500,6 +517,7 @@ local function set(conf_header_type, found_header_type, proxy_span, conf_default
   end
 
   if conf_header_type == "ot" or found_header_type == "ot" then
+    to_ot_trace_id = to_ot_trace_id or require "kong.plugins.opentelemetry.otlp".to_ot_trace_id
```

Comment:
moved here otherwise requiring "propagation" from "instrumentation" was resulting in `otlp` to be initialized too soon